packages/code-studio/src/settings/SettingsMenu.jsx

```diff
@@ -24,8 +24,15 @@ export class SettingsMenu extends Component {
     }
   }
 
+  static getVersion() {
+    return process.env.REACT_APP_VERSION;
+  }
+
   static handleExportSupportLogs() {
-    exportLogs();
+    const metadata = {
+      uiVersion: SettingsMenu.getVersion(),
```

Comment:
```suggestion
      uiVersion: SettingsMenu.getVersion(),
      userAgent: navigator.userAgent,
```

I didn't test that, but grabbing the user agent would be handy.